companies/services/kis_quote.py

```
@@ -97,6 +101,31 @@ def _get_access_token(self) -> Optional[str]:
             logger.error(f"KIS 토큰 발급 실패: {e}")
             return None
 
+    def _wait_for_rate_limit(self) -> None:
+        """
+        분산 환경에서 KIS API rate limit 준수를 위한 전역 딜레이 관리
+
+        Redis를 사용하여 마지막 요청 시간을 공유하고,
+        필요 시 딜레이를 적용하여 초당 2회 제한을 준수
+        """
+        import time
```

Ruff (0.14.13)

Redefinition of unused `time` from line 8: `time` redefined here

Remove definition: `time`

(F811)